<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModifyPTermial</class>
 <widget class="QDialog" name="DialogModifyPTermial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置连接点</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>32</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>连线方向：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>3</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>逻辑端号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditTerminalName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>3</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_dir">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>27</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>向左</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>向右</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>向上</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>向下</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="BtnOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>坐标</string>
   </property>
   <widget class="QPushButton" name="btn_GetTerminalPoint">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>拾
取</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditTerminalPointX">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>17</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditTerminalPointY">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>47</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>191</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string>连接点属性字库</string>
   </property>
   <widget class="QPushButton" name="BtnPutAttrDef">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>放 置</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableAttrDef">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>18</y>
      <width>191</width>
      <height>110</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>连接点属性</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_NoLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>内部端</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
